lib/src/main/java/com/thenewboston/data/dto/bankapi/DateSerializer.kt

```diff
@@ -0,0 +1,28 @@
+package com.thenewboston.data.dto.bankapi
+
+import kotlinx.datetime.Instant
+import kotlinx.datetime.LocalDateTime
+import kotlinx.datetime.TimeZone
+import kotlinx.datetime.toLocalDateTime
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+
+object DateSerializer : KSerializer<LocalDateTime> {
```

Comment:
Can you add a Unit Test for this?
Also let's stick with classes

Reply:
Yes

Reply:
I don't have merge access @MrIceman

Reply:
@MrIceman Unit Tests have been added and it has also been changed from an object into a class